gsy-node/modules/orderbook-worker/src/benchmarking.rs

```diff
@@ -13,106 +13,106 @@ use frame_system::RawOrigin;
 use frame_support::{sp_runtime::traits::{Hash, One}};
 use sp_std::vec;
 
-fn add_user<T: Config>(
```

Comment:
So you just commented it out in order to not to having to adapt it because it is "just" benchmarking?

Reply:
Exactly, I plan to reintroduce it in a separate PR now that the dependencies are in good shape